src/main/java/com/example/TicketBookingApp/exception/BookingTimeException.java

```diff
@@ -0,0 +1,7 @@
+package com.example.TicketBookingApp.exception;
+
+public class BookingTimeException extends RuntimeException{
```

Comment:
z ciekawości dlaczego wybrałeś `RuntimeException`?

Reply:
Dlatego, że są to wyjątki typu _unchecked_, nie wymagają one obsługi w kodzie i nie powodują przerwania pracy programu. Miały raczej na celu poinformować użytkownika jaki błąd wystąpił.